src/capacitor/index.ts

```diff
@@ -96,9 +96,9 @@ export class FluxCapacitor extends EventEmitter {
       .then(() => query);
   }
 
-  resize(pageSize: number, offset?: number): Promise<Results> {
+  resize(pageSize: number, resetOffset?: boolean): Promise<Results> {
     this.query.withPageSize(pageSize);
-    const skip = Math.floor((offset / pageSize)) * pageSize;
+    const skip = (resetOffset) ? 0 : Math.floor((this.page.fromResult / pageSize)) * pageSize;
```

Comment:
extra parentheses unnecessary:

`(resetOffset)`
`Math.floor((this.page.fromResult / pageSize))`